app/src/androidTest/java/com/github/h3lp3rs/h3lp/signin/GoogleSignInTest.kt

```diff
@@ -56,6 +57,9 @@ class GoogleSignInTest : H3lpAppTest() {
         val signInMock = mock(SignInInterface::class.java)
         When(signInMock.isSignedIn()).thenReturn(false)
 
+        val userSignIn = Storages.storageOf(Storages.SIGN_IN)
```

Comment:
You can import statically storageOf and SIGN_IN;)

Reply:
SIGNIN can't be statically imported

Reply:
use this : import com.github.h3lp3rs.h3lp.storage.Storages.SIGN_IN